### PR TITLE
Issue #KN-304 fix: Content players V2 (PDF) Web component output updation  

### PR DESCRIPTION
**Pull request by santoshilimi**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/KN-304" title="KN-304" target="_blank">RFCKN-304</a>  Content players V2 (PDF) Web component output updation
  </td></table>
  <br />
 

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ version: 2.1
 jobs:
   build:
     working_directory: ~/sunbird-pdf-player
-    docker:
-      - image: circleci/node:12.20.2-browsers
+    machine:
+      image: ubuntu-2004:202010-01
```
Comment (HarishGangula) [inline .circleci/config.yml:6]: Should use the circleci docker only

File: web-component/index.html [hunk 2]
```diff
@@ -26,6 +27,9 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"
       integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg=="
       crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/reflect-metadata/0.1.13/Reflect.min.js"
```
Comment (HarishGangula) [inline web-component/index.html:30]: Update the documentation for the changes

**Review by HarishGangula — APPROVED** [review]

File: web-component/index.html (hunk 2 above)
Comment (santoshilimi) [reply web-component/index.html:30]: Updated

File: .circleci/config.yml (hunk 1 above)
Comment (santoshilimi) [reply .circleci/config.yml]: Done

**Pull request merged**